api/utils/cron.js

```diff
@@ -0,0 +1,9 @@
+const CronJob = require('cron').CronJob;
+const DatabaseBackup = require('../db/backup.js');
+
+// Configure cron job to run every day at 12:00am
+let job = new CronJob('00 00 00 * * *', function () {
+  DatabaseBackup.createBackup();
+});
+
+job.start();
```

Comment:
Can we only start the cron job in production? I don't think we want this running on any other environment.